features/command_line_interface/exit_status.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Return the correct exit status codes
+  In order to use Rubycritic in Continuous Integration environments
+  Rubycritic returns the exit status code properly
+
+  Scenario: Status indicates a success when analyse a clean file
+    Given the clean file 'clean.rb'
+    When I run rubycritic clean.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates a success when analyse a smelly file
+    Given the smelly file 'smelly.rb'
+    When I run rubycritic smelly.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates a success when analyse a clean file with the minimum score option
+    Given the clean file 'clean.rb'
+    When I run rubycritic --minimum-score 100 clean.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates an error when analyse a smelly file with the minimum score option
+    Given the smelly file 'smelly.rb'
+    When I run rubycritic --minimum-score 100 smelly.rb
+    Then the exit status indicates an error
```

Comment:
I should have read the whole PR first ;) - this is an duplication to features/command_line_interface/minimum_score.feature (which you probably are aware of). I can see why you would do that but I still think it's redundant. I'd rather scrap this file completely and just use features/command_line_interface/minimum_score.feature. That you then can use this for CI is kind of implicitly.